### PR TITLE
Move to using svlogd for logs

### PR DESCRIPTION
**Pull request by tomdee**
Add --log-dir option to calicoctl

All services write to stdout/stderr and the logs are picked up my svlogd.

* This makes writing services easy (writing to stdout/err is a lower common denominator).
* This allows people who deploy calico-node to configure logging in a standard way across our components (to filter logs, change level or send logs to syslog)

I've also added a doc describing this docs/logging.md. I suggest starting there.

### CODE REVIEW

**Comment by spikecurtis** [comment]
Does svlogd further tie us to runit?

Does it make it easier to hook into everyone's favorite new/cool logging services?

File: node_filesystem/etc/service/bird/log/run [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+LOGDIR=/var/log/calico/bird
+mkdir -p $LOGDIR
+# Prefix each line with a timestamp
```
Comment (spikecurtis) [inline node_filesystem/etc/service/bird/log/run]: Doesn't bird already timestamp the log lines?

**Comment by spikecurtis** [comment]
One comment, but otherwise LGTM

File: node_filesystem/etc/service/bird/log/run (hunk 1 above)
Comment (tomdee) [inline node_filesystem/etc/service/bird/log/run]: Not when it's just logging to stdout.

**Pull request merged**